packages/manager/sms/src/sms/compose/addPhonebookContact/telecom-sms-sms-compose-addPhonebookContact.controller.js

```diff
@@ -41,16 +53,16 @@ export default class {
       hasBeenAdded: false,
     };
     this.selectedContacts = angular.copy(this.phonebooks.lists);
-    this.availableTypes = _.get(this.constant.SMS_PHONEBOOKS, 'numberFields');
+    this.availableTypes = get(this.constant.SMS_PHONEBOOKS, 'numberFields');
     this.$scope.$watch('AddPhonebookContactCtrl.phonebookContact.sorted', (contacts) => {
-      _.each(contacts, (contact) => {
-        const alreadyAdded = _.find(this.selectedContacts, { id: contact.id });
+      each(contacts, (contact) => {
```

Comment:
`each` -> `forEach`

Reply:
Interesting, the migration guide doesn't ask to do this: https://github.com/lodash/lodash/wiki/Changelog

It does make sense though